.github/workflows/ci.yml

```diff
@@ -49,7 +49,7 @@ jobs:
           sudo apt-get install -y ninja-build
         fi
 
-        sudo python3 -m pip install --upgrade pip
+        pip3 -V || sudo python3 -m pip install --upgrade pip
```

Comment:
Why is that needed?

Reply:
When CI stopped working, it was the solution. Not to install newer version if current still works. Anyway, version 19 is ok, and we don't need to spend time installing 20-th version.